cedar-policy/src/api.rs

```diff
@@ -2269,6 +2270,17 @@ pub enum PolicySetError {
     /// Error when removing a link that is not a link
     #[error("unable to unlink `{0}` because it is not a link")]
     UnlinkLinkNotLinkError(PolicyId),
+    /// Error when converting from EST
+    #[error("Error deserializing an EST policy/template: {0}")]
+    #[diagnostic(transparent)]
+    FromEST(#[from] cedar_policy_core::est::FromJsonError),
+    /// Error when converting to EST
+    #[error("Error serializing a policy to the EST format: {0}")]
+    #[diagnostic(transparent)]
+    ToEST(#[from] PolicyToJsonError),
```

Comment:
Similarly here we can avoid using the "EST" internal terminology and just say `FromJson` and "Error deserializing a JSON policy/template" and "Error serializing a policy to the JSON format" etc